strings/base_string.h

```diff
@@ -638,3 +638,27 @@ WINRT_EXPORT namespace winrt
         return result;
     }
 }
+
+namespace winrt::impl
+{
+    template <size_t Size>
+    hstring literal_to_hstring(std::array<char_type, Size> const& value) noexcept
+    {
+        return to_hstring(std::string_view(reinterpret_cast<char const*>(value.data()), Size - 1));
```

Comment:
Recommend adding an overload of `to_hstring` for `std::u8string_view`, which will reduce developer friction ~~if~~ when they use `char8_t` strings. Then, the `reinterpret_cast` can live there, as its only purpose is to translate from `char8_t` to the `char` required by `MultiByteToWideChar`.

Reply:
Yeah, I was hoping to use `u8string_view` but it looks like its new to C++20.

Reply:
`u8string_view` is C++to, but `basic_string_view<winrt::impl::char_type>` always works.